composer.json

```diff
@@ -21,25 +21,24 @@
     ],
     "require": {
         "composer/installers": "^1.2",
-        "cweagans/composer-patches": "^1.6",
-        "drupal-composer/drupal-scaffold": "^2.2",
+        "cweagans/composer-patches": "^1",
+        "drupal-composer/drupal-scaffold": "^2",
         "drupal/config_direct_save": "^1.0",
-        "drupal/config_installer": "^1.0",
-        "drupal/config_provider": "2.0-alpha1",
-        "drupal/config_readonly": "^1.0@beta",
-        "drupal/config_sync": "2.0-alpha4",
+        "drupal/config_provider": "^2",
+        "drupal/config_readonly": "^1",
+        "drupal/config_sync": "^2",
         "drupal/console": "^1",
-        "drupal/core": "~8.0",
-        "drush/drush": "~8.0|^9.0.0-beta7",
+        "drupal/core": "^8",
```

Comment:
https://github.com/zaporylie/composer-drupal-optimizations#configuration requires `drupal/core: ^8.5` or above. Any reason not to set that here just in case?

Reply:
I guess we could makes sense, but the drupal-optimization would require that automatically

Reply:
It looks like that dependency was intentionally removed from `composer-drupal-optimizations` https://github.com/zaporylie/composer-drupal-optimizations/pull/6. I've updated this in 0435f8cb979c6516a692864b95ff38555dacb3d0. Folks can always roll back if needed, but I can't see much reason to on a fresh build.